packages/labs/context/src/test/provider-and-consumer_test.ts

```diff
@@ -21,7 +22,17 @@ class ContextConsumerAndProviderElement extends LitElement {
   @property({type: Number})
   public value = 0;
 
+  @provide<string, any>({context: stringContext})
```

Comment:
Needing to pass in the `any` type is a big code smell, what about instead an alias of the provide function with looser typing for private fields? We could call it `@providePrivate`

Reply:
I commented on the issue too, but my feeling is just making `provide` looser seems better considering the eventual upgrade to standard decorators that wouldn't need this workaround.

Reply:
Oh, standard decorators give the decorator function's type access to the type of the decorated field?

Reply:
It is mentioned here: https://github.com/lit/lit/issues/3883#issuecomment-1541007298

Which does appear to be the case if I'm reading this correctly:
https://github.com/microsoft/TypeScript/pull/50820#:~:text=to%20support%20correctly.-,Type%20Checking,-When%20a%20decorator